src/distributions/pointmass.jl

```diff
@@ -19,6 +19,7 @@ variate_form(::PointMass{M}) where {T, M <: AbstractMatrix{T}} = Matrixvariate
 sampletype(distribution::PointMass{T}) where {T} = T
 
 getpointmass(distribution::PointMass) = distribution.point
+getpointmass(point::Any) = point
```

Comment:
Might this cause issues when another distribution comes in? such as 
```julia
getpointmass(point::MvNormalMeanPrecision) = point
```
Would it instead be sufficient to have
```julia
getpointmass(::Real)
getpointmass(::AbstractArray)
```

Reply:
We discussed this with @wmkouw , but I think @bartvanerp has a valid concern. I didn't think of the `MvNormalMeanPrecision`. Perhaps it is better to constraint it to `Union{Real, AbstractArray}` indeed.

Reply:
Cool. I will update it.